scrape/target.go

```diff
@@ -459,47 +459,27 @@ func PopulateLabels(lb *labels.Builder, cfg *config.ScrapeConfig, noDefaultPort
 
 	// addPort checks whether we should add a default port to the address.
 	// If the address is not valid, we don't append a port either.
```

Comment:
Iiuc this is comment line is no longer accurate?

Reply:
Yes, the comment needs an update.

Reply:
But maybe we don't need the whole logic anymore. Let me investigate.